server/routes/user_badges.js

```diff
@@ -3,7 +3,7 @@ const User = require('../models/user');
 
 
 const router = new Router({
-  prefix: '/leaderboard'
+  prefix: '/user_badges'
 });
 
 router.get('/', async ctx => {
```

Comment:
Kindly provide a way to filter badges earned this week if this endpoint will also serve as the leader-board API

Reply:
@mimidotsuser I'm working on this